pkg/registry/core/pod/storage/eviction.go

```diff
@@ -291,14 +292,19 @@ func (r *EvictionREST) Create(ctx context.Context, name string, obj runtime.Obje
 	}
 
 	// Try the delete
-	err = addConditionAndDeletePod(r, ctx, eviction.Name, rest.ValidateAllObjectFunc, deleteOptions)
+	err = addConditionAndDeletePod(r, ctx, pod, deleteOptions)
```

Comment:
revert the signature change here, we're not using the looked-up pod at all any more inside addConditionAndDeletePod